task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -1548,7 +1548,7 @@ def execute(self, context):
             log=mock.ANY,
         )
 
-    @pytest.mark.parametrize("value", [VALUE, 0])
+    @pytest.mark.parametrize("value", [VALUE, 0, (1, 2)])
```

Comment:
This feels somewhat like testing by side effect - given the test name is params we should see if there is a more explicit place to test